modules/couchdb/collect.go

```diff
@@ -139,6 +145,19 @@ func aggregateHTTPStatusCodes(collected map[string]int64, metric string, value i
 	}
 }
 
+func findMaxMQSize(MessageQueues map[string]interface{}) int64 {
+	var max float64
+	for _, mq := range MessageQueues {
+		switch mqSize := mq.(type) {
+		case float64:
+			max = math.Max(max, mqSize)
+		case map[string]interface{}:
+			max = math.Max(max, mqSize["count"].(float64))
```

Comment:
this panics if `map[string]interface{}` has no `count` key or `mqSize["count"]` value is not `float64`

Reply:
fix is
```go
if v, ok := mqSize["count"].(float64); ok {
	...
}
```

Reply:
Fixed